### ISSUE TITLE
'buildah bud --pull-always .' not always pulling

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [projectatomic/buildah#281]
The pull-always argument for 'buildah bud' seems to not be working.  The below example shows that a pull is performed  on the first invocation, but follow on invocations seem to be using the local image rather than pulling anew.

Fedora 26

```
# buildah bud --pull-always .
STEP 1: FROM alpine
Getting image source signatures
Copying blob sha256:88286f41530e93dffd4b964e1db22ce4939fffa4a4c665dab8591fbab03d4926
 1.90 MiB / 1.90 MiB [=========================================================]
Copying config sha256:76da55c8019d7a47c347c0dceb7a6591144d232a7dd616242a367b8bed18ecbc
 0 B / 1.48 KiB [--------------------------------------------------------------]
Writing manifest to image destination
Storing signatures
STEP 2: CMD ls -alF
STEP 3: COMMIT containers-storage:[overlay@/var/lib/containers/storage+/var/run/containers/storage]@a81f6e1a396573237f39a216767e864567b1a7fdc0766ceaa3cee70695345811
 1.48 KiB / 1.48 KiB [=========================================================]
# 
# buildah bud --pull-always .
STEP 1: FROM alpine
STEP 2: CMD ls -alF
STEP 3: COMMIT containers-storage:[overlay@/var/lib/containers/storage+/var/run/containers/storage]@3ea22c12c122da6caf18d6cc1398a13ef9e1e13c1f97f2979292ba79aa74a743
[root@localhost cri-o]# cat Dockerfile
FROM alpine 
CMD ls -alF
#
# buildah version
Version:       0.3
Go Version:    go1.8.1
Image Spec:    1.0.0
Runtime Spec:  1.0.0
Git Commit:    e9748f0
Built:         Thu Jul 20 16:45:03 2017
OS/Arch:       linux/amd64

```

**Comment by rhatdan** [comment]
Good, maybe we can remove it. Not sure what the use case is for it.  Also not sure what the use case for --pull=false is

**Comment by rhatdan** [comment]
@nalind WDYT?

**Comment by TomSweeneyRedHat** [comment]
This was fixed as a by product of #419.
